test/proto/test.proto

```diff
@@ -71,3 +71,11 @@ service TestService {
   rpc HalfDuplexCall(stream StreamingOutputCallRequest)
       returns (stream StreamingOutputCallResponse);
 }
+
+
+// A simple service NOT implemented at servers so clients can test for
+// that case.
+service UnImplementedService {
```

Comment:
total nit: shouldn't we take "unimplemented" as a single word and thus remove the capital I?